docs/installation/kind.md

```diff
@@ -16,39 +16,72 @@ KIND (Kubernetes in Docker) deployment of OVN kubernetes is a fast and easy mean
 - [kubectl]( https://kubernetes.io/docs/tasks/tools/install-kubectl/ )
 - Python and pip
 - jq
+- openssl
```

Comment:
oh they were needed on KIND? I don't remember having to install these two..but I did this a long time ago

Reply:
Just did some replication testing to confirm. SSL is mentioned directly in the kind.sh script as a dependency but it doesnt get installed automatically. Not installing Openvswitch causes the kind.sh script to fail to execute. I believe what happens is the rollout requires each node to bring up their dataplane with ovs but kind.sh does not have a dependency check to install openvswitch if not present on the host.